vcpkg.json

```
@@ -15,7 +15,9 @@
     "prometheus-cpp",
     "ftxui",
     "rocksdb",
-    "boost-property-tree"
+    "boost-property-tree",
+    "openssl",
```

`liblsquic` requires `boringssl`.
Don't need to change `find_package(OpenSSL)`
```suggestion
    "boringssl",
```